src/demos/demo.service.ts

```diff
@@ -431,6 +432,22 @@ export class SkyDemoService {
         }
       ]
     },
+    {
+      name: 'Fuzzy datepicker',
```

Comment:
Should this really be a separate entry from the datepicker demo?

Reply:
Yeah, we actually have a pattern of this with other demos (filters). Keeping the separate helps us isolate changes and keep these modular.

Reply:
Good deal. I wasn't sure. I'll keep that in mind in the future. Thanks!